core/src/main/java/io/ultrabrew/metrics/data/BasicHistogramAggregator.java

```diff
@@ -41,8 +41,8 @@
 public class BasicHistogramAggregator extends ConcurrentIntTable implements Aggregator {
```

Comment:
as discussed in person with @smrutilal2, we should not absorb the functionality of gauge/timer into histogram aggregator, rather let them run in parallel. this will allow users to eventually have multiple aggregators depending on use cases. eg, timer+histogram+t-digest+HLL for a metric but different downstream use cases.